### PR TITLE
Treatment notes

### PR DESCRIPTION
**Pull request by JohnColvin**
- [x] response data
- [x] description of content schema
- [x] allowed html in answers

### CODE REVIEW

**Comment by bpinto** [comment]
Perhaps we could add to the creation example a very basic content?

**Comment by JohnColvin** [comment]
Added @bpinto

File: sections/treatment_notes.md [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+  "id": 1,
+  "created_at": "2017-06-12T16:56:57Z",
+  "updated_at": "2017-06-12T16:56:57Z",
+  "content": {
+    "sections": [
+      {
+        "name": "Section 1",
+        "questions": [
+          {
+            "name": "Question 1,
```
Comment (Billiam) [inline sections/treatment_notes.md]: Missing closed quote

Comment (JohnColvin) [inline sections/treatment_notes.md]: Fixed. Thanks! I guess all the red highlighting should have been a clue 🤦‍♂️

**Pull request merged**